From 67c8a4741224fd3d64a5d26a569a2240d894b502 Mon Sep 17 00:00:00 2001
From: Alan Shearer <sakison@gmail.com>
Date: Sun, 11 Oct 2026 12:30:45 +0000
Subject: [PATCH] AboutSystem: update FreeBSD and ACPICA copyright years

Update AboutSystem's old copyright years for FreeBSD and ACPICA.
Added mention of various network drivers, since we have a bunch of those from FreeBSD in the tree.
This only touches the base system, not HaikuPorts packages.
---
 src/apps/aboutsystem/AboutSystem.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/apps/aboutsystem/AboutSystem.cpp b/src/apps/aboutsystem/AboutSystem.cpp
index 2b59c4e1..635a5fb8 100644
--- a/src/apps/aboutsystem/AboutSystem.cpp
+++ b/src/apps/aboutsystem/AboutSystem.cpp
@@ -1769,8 +1769,8 @@ AboutView::_CreateCreditsView()
 	AddCopyrightEntry("The FreeBSD Project",
 		B_TRANSLATE("Contains software from the FreeBSD Project, "
 		"released under the BSD license:\n"
-		"ftpd, ping, telnet, telnetd, traceroute\n"
-		COPYRIGHT_STRING "1994-2008 The FreeBSD Project. "
+		"ftpd, ping, telnet, telnetd, traceroute, and various network drivers\n"
+		COPYRIGHT_STRING "1994-2024 The FreeBSD Project. "
 		"All rights reserved."),
 		StringVector(kBSDTwoClause, kBSDThreeClause, kBSDFourClause,
 			NULL),
@@ -1890,7 +1890,7 @@ AboutView::_CreateCreditsView()
 
 	// acpica copyrights
 	_AddPackageCredit(PackageCredit("ACPI Component Architecture (ACPICA)")
-		.SetCopyright(COPYRIGHT_STRING "1999-2018 Intel Corp.")
+		.SetCopyright(COPYRIGHT_STRING "1999-2026 Intel Corp.")
 		.SetLicense("Intel (ACPICA)")
 		.SetURL("https://www.intel.com/content/www/us/en/developer/topic-technology/open/acpica/overview.html"));
 
-- 
2.39.5
